apps/core/components/ItineraryCard/SectorBorder.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+
+import * as React from 'react';
+import { StyleSheet, Badge } from '@kiwicom/universal-components';
+import { View } from 'react-native';
+import * as DateFNS from 'date-fns';
+
+import ItineraryCardRow from './ItineraryCardRow';
+
+type Props = {|
+  from: ?Date,
+  to: ?Date,
+|};
+
+const getDuration = (arrivalTime, departureTime) => {
+  return DateFNS.distanceInWords(departureTime, arrivalTime);
+};
+
+export default function SectorBorder({ from, to }: Props) {
+  return (
+    <ItineraryCardRow>
+      <View style={styles.leftShift}>
+        <Badge type="neutral">Returns in {getDuration(from, to)}</Badge>
```

Comment:
This will work only for `return` type. Maybe we should start with more universal `X nights in destination` (skip `then depart from` for now) which is used even for return type on web and then extend it with this `Return in ...` scenario if necessary.

<img width="355" alt="screenshot 2019-02-11 at 16 19 59" src="https://user-images.githubusercontent.com/2660330/52573120-b78c5c00-2e19-11e9-9801-389c1c7794c9.png">

Reply:
According to the newest design it should be "Returns in ..." (even for nomad), but you are right, it's not universal. I will fix it.

Reply:
Fixed with using prefix "Stays" as we agreed with @JoseDuSV